client/src/modules/chapters/pages/chapterPage.tsx

```diff
@@ -52,6 +60,10 @@ export const ChapterPage: NextPage = () => {
         <Text fontSize={'lg'} color={'gray.500'}>
           {data.chapter.description}
         </Text>
+        <Heading size="md" color={'gray.700'}>
+          Chat Link:
+        </Heading>
+        <Link>{data.chapter.chatUrl}</Link>
```

Comment:
Could you make this conditional on the chatUrl existing?  It looks a little odd to see Chat Link: without the link.